src/pages/experiments/[experimentId]/plots-and-tables/marker-heatmap/index.jsx

```diff
@@ -59,9 +59,22 @@ const MarkerHeatmap = ({ experimentId }) => {
 
   useEffect(() => {
     if (louvainClustersResolution && config) {
-      dispatch(loadMarkerGenes(experimentId, louvainClustersResolution, plotUuid, config.numGenes));
+      dispatch(loadMarkerGenes(
+        experimentId, louvainClustersResolution, plotUuid, config.numGenes, config.selectedCellSet,
+      ));
     }
-  }, [louvainClustersResolution, hierarchy, config?.numGenes]);
+  }, [louvainClustersResolution, config?.selectedCellSet, config?.numGenes]);
```

Comment:
do we have to handle `loadMarkerGenes` with custom cellsets in the data exploration heatmap somehow? or is this something only for the plots and tables one?

Reply:
No we don't need to handle it - there is no menu to choose data so we use only louvain

Reply:
I saw its fixed in master so i removed this change